### ISSUE TITLE
Singleplayermod: Getting Damage

### ISSUE THREAD

**Issue by JAKAMI99** [sailro/EscapeFromTarkov-Trainer#188]
When using the Trainer, I get random damage while shooting my gun. Died (KIA) 3 rounds in a Row while shooting.
Is this an anticheat measurement implemented in the client?
Fascinating and confusing at the same time :D

**Comment by sailro** [comment]
Weird, never saw that before. What is your setup? (EFT/spt-aki versions) ?

**Comment by JAKAMI99** [comment]
Sorry for my late reply, I was busy yesterday.
I downgraded the latest EFT version to the latest one which is compatible with spt-aki, so:
EFT-Version: 12.12.15.17349
SPT-AKI-Version: 2.3.1

When I use the godmode cheat “health”, I only hear the hits I get, but don't die.

I attached a clip where this weird thing happens:
[Recording](https://youtu.be/BoVYIUo6PPk)
**(Highlights are marked as chapters)**

The first encounter is at 02:39 where I was able to kill him at 03:50
And the final death is also due to self damage at 04:40

**Comment by sailro** [comment]
Yep, I think it is related to this specific map `Interchange`. They made a lot of tweaks given it's huge. I just tried with `Factory` and I'm not able to repro with the latest EFT and spt-aki 3.0.0

**Comment by JAKAMI99** [comment]
Yes, one to change the server settings like spawn probability of items and the skill gain multipier.

**Comment by sailro** [comment]
I think it is probably the same thing as https://github.com/sailro/EscapeFromTarkov-Trainer/issues/174

We always had incompatibilities with those modes.

**Comment by JAKAMI99** [comment]
Wouldn't say it's the same. It's really weird that it's the exact same damage projected to my player model. Shots on the torso get projected  even tho the armor gets calculated even when the enemy is wearing none. Gonna try to test it more and come back when I know more

**Comment by laniidae** [comment]
i believe it's related to specific type of ammo
